src/ExampleIsm.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+import { IInterchainSecurityModule } from "@hyperlane-xyz/contracts/interfaces/IInterchainSecurityModule.sol";
+
+contract ExampleIsm is IInterchainSecurityModule {
+    // @dev The first 128 enum values are reserved for core Hyperlane types
+    uint8 public constant COMMUNITY_MASK = 0x80;
+
+    // TODO: Replace with actual community types you want to use
+    enum CommunityTypes {
+        EXAMPLE_ISM
+    }
+
+    // @inheritdoc IInterchainSecurityModule
+    uint8 public moduleType = uint8(CommunityTypes.EXAMPLE_ISM) | COMMUNITY_MASK;
```

Comment:
kinda clever but doesnt really leave room for migration? in general im not a huge fan of the enum based identifiers
I believe @nambrot is thinking that most of the ISMs in here will actually be CCIP-Read ISM type

Reply:
ok, let's chat about this in protocol meeting

Reply:
Agree with @yorhodes 's cmment

Reply:
done